src/exoscale/interceptor/impl.cljc

```diff
@@ -8,6 +8,10 @@
   #?(:clj clojure.lang.IPersistentMap
      :cljs cljs.core.PersistentHashMap)
   (interceptor [m] (map->Interceptor m))
+  
+  #?(:clj clojure.lang.IPersistentArrayMap
```

Comment:
I don't think it's necessary for :clj

Reply:
as a matter of fact CI blows up on it.

Reply:
Oop, my bad.  May have got the type name wrong.  Pretty sure CLJ also has a concrete ArrayMap type that can be used here... though ig it doesn't matter since it'll be caught by general ILookup case below.

Reply:
Oh I see.  It has `IPersistentMap` for Clojure, not the concrete type.  Missed that, my bad.  I'll remove the `:clj` portion from the PR when I get the chance.